guides/troubleshooting/inotify-watch-limits.md

```diff
@@ -0,0 +1,275 @@
+---
+title: inotify Watcher Limit Problems
+description: Learn how to resolve problems related to the inotify Watcher Limit.
+---
+
+When using some applications and tools, including Webpack or [code-server], you
+may encounter an error similar to the following:
+
+> Watchpack Error (watcher): Error: ENOSPC: System limit for number of file
+> watchers reached, watch '/some/path'
+
+[code-server]: https://github.com/cdr/code-server
+
+This article will show you how to diagnose and troubleshoot this error, which
+relates to an elevated number of inotify watchers in use.
+
+## Background
+
+[`Inotify`] allows programs to monitor files for changes, so that they receive
```

Comment:
I think we should use lowercase `inotify` here, even though it's at the start of the sentence, since it refers to a system call